unix_commands.sh

```diff
@@ -0,0 +1,14 @@
+echo Shaked >> a-1
+touch 1-b
+echo Shaked >> 1-b
+mkdir new
+cp 1-a new/
+mv 1-a 1-a-new
+gedit 1-a-new
```

Comment:
instead of using gedit, consider using a command-line text editor like nano to edit the file 1-a-new. This will allow the script to continue executing without requiring manual input or waiting for the text editor to be closed.

Reply:
The task required that we use the gedit editor.